assets/manifests.json

```diff
@@ -0,0 +1,5 @@
+{
+    "awsHello": "file:example-aws-hello.json",
+    "zendesk": "0.0.1",
```

Comment:
why is it not "com.mattermost.zendesk"? (or do we support both)

Reply:
ATM, the top-level command is the same as appid. If we decouple them, the app_id is just a (unique) string. I'm 0/5 on the semantics and conventions.